packages/cozy-harvest-lib/src/components/KonnectorConfiguration/ConfigurationTab/index.jsx

```diff
@@ -30,6 +33,7 @@ const ConfigurationTab = ({
   flow
 }) => {
   const { t } = useI18n()
+  const { isMobile } = useBreakpoints()
```

Comment:
It's a breaking change in Harvest since we have to use the `BreakpointsProvider` in the consuming app, right?

Reply:
Correct, although if I am correct, only home and banks are using Harvest. cozy-banks already has the BreakpointsProvider, and I am adding it in cozy-home.

Nonetheless I have edited the commit to add the BREAKING CHANGE mention 👍